includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -141,6 +141,38 @@ public function register() {
 		add_action( 'enqueue_block_assets', [ $this, 'enqueue_editor_assets' ] );
 		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_assets' ] );
 		add_action( 'edit_form_before_permalink', [ $this, 'register_generated_titles_template' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'admin_enqueue_script' ] );
```

Comment:
We already have a function hooked to `admin_enqueue_scripts` (see line 142 above). Thinking instead of having duplicate hooks, we keep just the one we already have (`enqueue_admin_assets`). If we want to keep that method cleaner, instead of having this new functionality added to that, we could take the existing code and the new code, put those both in their own methods and then our `enqueue_admin_assets` method could call those both.